.github/actions/stage2-setup/action.yml

```diff
@@ -98,6 +98,8 @@ runs:
         if [ "$SATELLITE_NAME" = "core-test" ]; then
           SATELLITE_NAME="$SATELLITE_NAME-$(( $RANDOM % 4 ))"
           echo "Using core-test satellite: $SATELLITE_NAME"
+          # limit converstion parallelism to prevent overloading the core-test satellite
+          ${{inputs.SUDO}} ${{inputs.BUILT_EARTHLY_PATH}} config global.conversion_parallelism 2
```

Comment:
Couldn't we just limit this on the satellite side?

Reply:
we've tried that too, might make sense to just lower the parallelism again on the satellite size.